cmd/monta/main.go

```diff
@@ -423,6 +424,54 @@ func newListWalletTransactionsCommand() *cobra.Command {
 	return cmd
 }
 
+func newStartChargeCommand() *cobra.Command {
+    cmd := &cobra.Command{
+        Use:   "start-charge",
+        Short: "Start a charge session",
+        Annotations: map[string]string{
+            apiCommandAnnotation: "POST /v1/charges",
+        },
+    }
+    cmd.SetHelpFunc(helpFunc)
+    cmd.SetUsageFunc(usageFunc)
+    cmd.SetOut(os.Stdout)
+    cmd.SetErr(os.Stderr)
+    payingTeamID := cmd.Flags().Int64("paying-team-id", 0, "paying team ID to pay for the charge")
+    cmd.Flag("paying-team-id").Annotations = map[string][]string{
+        argumentFlagAnnotation: {},
+    }
+    chargePointID := cmd.Flags().Int64("charge-point-id", 0, "charge point ID to start charge")
+    cmd.Flag("charge-point-id").Annotations = map[string][]string{
+        argumentFlagAnnotation: {},
+    }
+    reserveCharge := cmd.Flags().Bool("reserve-charge", false, "reserve charge determines to start or reserve charge")
```

Comment:
Does one not always start a charge when sending this commend, then also reserving it is optional, the bool.

Reply:
This should be a one-to-one mapping with the API to the furthest extent possible, so with that in mind I think what we have here makes sense - with the addition that we should copy the API documentation straight into the flag comments - that way we can leave it to the Monta team to wordsmith on the documentation :smile: 

![image](https://user-images.githubusercontent.com/1392777/194579875-1e72d0ac-be43-4675-8b42-54bb734893e0.png)